cmd2/cmd2.py

```diff
@@ -2380,13 +2576,17 @@ def get_help_topics(self) -> list[str]:
         # Filter out hidden and disabled commands
         return [topic for topic in all_topics if topic not in self.hidden_commands and topic not in self.disabled_commands]
 
-    def sigint_handler(self, signum: int, _: Optional[FrameType]) -> None:  # noqa: ARG002
+    def sigint_handler(
+        self,
+        signum: int,  # noqa: ARG002,
+        frame: Optional[FrameType],  # noqa: ARG002,
```

Comment:
`frame` is still unused right?  Or does it have to be called that?

Reply:
It bothered me that it wasn't named since it's a part of our public API.

Reply:
Do we need it as part of a signint handler?  Can we remove it? Alternatively we can call it `_frame`